src/hotkeys.cc

```diff
@@ -1265,11 +1265,19 @@ void handle_paging_key(int ch)
             }
             break;
 
+        case KEY_CTRL_B:
```

Comment:
This seems like it would be useful?  Could the hotkey be CTRL-F, it might be easier to remember.